tests/conftest.py

```diff
@@ -362,6 +362,7 @@ def _run_snake_oil(source_root):
         parser,
         [
             ENSEMBLE_EXPERIMENT_MODE,
+            "--disable-monitor",
```

Comment:
Is it `--disable-monitor`, or `--disable-monitoring`?
I see another integration test uses `...-monitoring`

Reply:
I guess we can use both, but should we have it consistent?

Reply:
thx, it should only be `monitoring`.

Reply:
`argparse` allows for shortcut on options..

Reply:
Fixed.